spec/std/signal_spec.cr

```diff
@@ -1,9 +1,10 @@
 {% skip_file if flag?(:wasm32) %}
 
-require "spec"
+require "./spec_helper"
 require "signal"
 
-describe "Signal" do
+# interpreted code never receives signals (#12241)
+pending_interpreted describe: "Signal" do
```

Comment:
**suggestion:** Wouldn't it make more sense to skip this entire file like on wasm?

Reply:
`skip_file` suggests signals won't be on WebAssembly at all, but it looks like there are multiple attempts at POSIX compatibility layers, so I think `skip_file` is the less appropriate one here (or rather, it hasn't been updated to use `pending_wasm32` after #14086)

Reply:
Could be.

My main concern is that I find `pending_interpreted describe: ""` over the entire contents of a file very unergonomic. It prevents grepping `describe Signal`, for example. And it's needles complexity.

I'd prefer to skip the file for wasm and interpreter and add comments to explain this might be enabled in the future.

Reply:
There are already two uses of `pending_wasm32 describe:`, so IMO this should be taken care of in a separate PR.